docs/cross-compilation.md

```diff
@@ -0,0 +1,63 @@
+# How to analyze a project that contains cross-compilation 
+
+It may happen that due to special gcc build-time configuration (see https://gcc.gnu.org/onlinedocs/libstdc++/manual/configure.html) clang cannot do the analysis.
+This is especially the case when gcc is used for **cross-compilation**.
+
+**In case of cross-compilation always use the same target for clang analysis as you use in the gcc build.** 
+
+A file (`commonhandler.cpp`) for example compiles like this to powerpc target:
+``` 
+powerpc64-wrs-linux-g++ -c -o /home/<username>/mylib_ppc64_linux/commonhandler.cpp
```

Comment:
I would recommend to use `powerpc64-linux-g++` as a compiler name and please update the include paths in the saargs and tidyargs examples too to reflect this change.